### PR TITLE
Wercker

### LINKED ISSUE #19: wercker!
deployyy

### PR DESCRIPTION
**Pull request by blisteringherb**
### What does this PR do?

Adds basic deployment functionality to the Gladiator app. We may need to re-add some artisan commands in the future.

### How do I test?

Merging into Master will automatically deploy the code to the QA Gladiator box, and builds can be manually deployed to the production gladiator box. Check out the Wercker app here - https://app.wercker.com/#applications/56cf5c7543defb820306eed4

Resolves #19

### CODE REVIEW

File: wercker.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+box: blisteringherb/scholarship@0.0.6
```
Comment (angaither) [inline wercker.yml]: scholarship?

Comment (blisteringherb) [inline wercker.yml]: Yeah, I should've named it differently a long time ago, probably something like dosomething-laravel, but it's a basic box to build laravel apps on.

Comment (angaither) [inline wercker.yml]: ah, okie

**Comment by angaither** [comment]
:+1:

**Pull request merged**